make/test.mk

```diff
@@ -80,6 +80,8 @@ olm-integration-setup: olm-integration-cleanup
 .PHONY: olm-integration-cleanup
 olm-integration-cleanup: get-test-namespace
 	$(Q)oc login -u system:admin
+	$(Q)-oc delete catalogsources -n olm my-catalog
```

Comment:
The above code should work. However, writing it like this would be better.
The syntax should be `$(Q)-oc delete catalogsource my-catalog -n olm`

(Same with subscription also)